enterprise/lms_api.py

```diff
@@ -24,36 +28,97 @@
 except ImportError:
     CourseEnrollment = None
 
+try:
+    from openedx.core.lib.token_utils import JwtBuilder
+except ImportError:
+    JwtBuilder = None
+
 
+LOGGER = logging.getLogger(__name__)
 LMS_API_DATETIME_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 
 
 class LmsApiClient(object):
     """
     Object builds an API client to make calls to the edxapp LMS API.
+
+    Authenticates using settings.EDX_API_KEY.
     """
 
-    def __init__(self, url, append_slash=False):
+    API_BASE_URL = settings.LMS_ROOT_URL + '/api/'
```

Comment:
I've been doing some work with the [settings for Enterprise](https://github.com/edx/edx-platform/pull/14762), and seeing all of these URL definitions in the client implementations with references to 'settings.LMS_ROOT_URL` makes me think we should be adding them to `common.py` with corresponding overrides in `aws.py`.

What are your thoughts on this?